components/vector-kubearchive-log-collector/production/stone-prod-p02/vector-helm-values.yaml

```diff
@@ -0,0 +1,165 @@
+---
+role: Agent
+resources:
+  requests:
+    cpu: 512m
+    memory: 4096Mi
+  limits:
+    cpu: 2000m
+    memory: 4096Mi
+customConfig:
+  data_dir: /vector-data-dir
+  api:
+    enabled: true
+    address: 127.0.0.1:8686
+    playground: false
+  sources:
+    k8s_logs:
+      type: kubernetes_logs
+      rotate_wait_secs: 5
+      glob_minimum_cooldown_ms: 500
+      max_line_bytes: 3145728
+      auto_partial_merge: true
+      extra_label_selector: "app.kubernetes.io/managed-by in (tekton-pipelines,pipelinesascode.tekton.dev)"
```

Comment:
Please make sure that the content of this file is exactly the same of `staging/stone-stg-rh01/vector-helm-values.yaml` and `development/vector-helm-values.yaml`. The three of them should be identical.